ci-operator/step-registry/assisted/vsphere/setup/template/assisted-vsphere-setup-template-ref.yaml

```diff
@@ -0,0 +1,33 @@
+ref:
+  as: assisted-vsphere-setup-template
+  from: assisted-test-infra-internal
+  commands: assisted-vsphere-setup-template-commands.sh
+  credentials:
+  - namespace: test-credentials
+    name: vsphere-vmc
+    mount_path: /var/run/vault/vsphere
+  - namespace: test-credentials
+    name: ci-ibmcloud
+    mount_path: /var/run/vault/ibmcloud
```

Comment:
where do we use those ibmcloud creds?

Reply:
We have two vsphere envs on AWS and on IBM Cloud.
The LEASED_RESOURCE controls what cluster to use.
please read: https://github.com/openshift/release/blob/6fcfd06ade2161c7dad325778072538ac9c0855a/ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh#L24